idex.html

```diff
@@ -11,208 +11,230 @@
+                        <li><img class="about__images" src="./images/img-languages-html.jpg" alt="html icon">HTML</li>
+                        <li><img class="about__images" src="./images/img-languages-css.jpg" alt="css icon">CSS</li>
+                    </ul>
                 </li>
                 <li>
-                    <label for=""></label>
-                    <input class="footer__input" type="email" id="mail" name="user_email" placeholder="Your email"
-                        required />
+                    <div class="knowledge__titles">Frameworks<span class="material-symbols-outlined">
+                            chevron_right
+                        </span>
+                    </div>
                 </li>
+                <hr>
                 <li>
-                    <label for=""></label>
-                    <textarea class="footer__input footer__textarea" id="msg" name="user_message"
-                        placeholder="Write your message here" maxlength="500" required></textarea>
+                    <div class="knowledge__titles">Skills<span class="material-symbols-outlined">
+                            chevron_right
+                        </span>
+                    </div>
                 </li>
+                <hr>
             </ul>
-            <button type="submit" class="footer__button" href=""><b>Get In Touch</b></button>
-        </form>
-    </footer>
+        </div>
+    </section>
+    <!-- Footer -->
+    <div class="footer__background">
+        <footer class="footer">
+            <h3 class="footer__title">Contact me</h3>
+            <p class="footer__paragraph">If you have an application you are interested in developing, a feature that you
+                need built or a project
+                that needs coding. I’d love to help with it </p>
+            <form class="footer__form" action="https://formspree.io/f/xwkjeknw" method="post">
+                <ul class="footer__ul">
+                    <li>
+                        <label for=""></label>
+                        <input class="footer__input" type="text" id="name" name="user_name" placeholder="Your name"
+                            maxlength="30" required />
+                    </li>
+                    <li>
+                        <label for=""></label>
+                        <input class="footer__input" type="email" id="mail" name="user_email" placeholder="Your email"
+                            required />
+                    </li>
+                    <li>
+                        <label for=""></label>
+                        <textarea class="footer__input footer__textarea" id="msg" name="user_message"
+                            placeholder="Write your message here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+                <button class="footer__button" type="submit" href="">Get In Touch</button>
+            </form>
+        </footer>
```

Comment:
- [ ] kindly update the background image for the contact-me section to match Figma design


![image](https://user-images.githubusercontent.com/89282221/213803884-27002ade-54eb-428e-a7bd-91a770f8b8ea.png)


Figma 
![image](https://user-images.githubusercontent.com/89282221/213803756-8133d436-e081-4b15-9a70-16f45f71db4a.png)